zds/tutorialv2/managers.py

```diff
@@ -26,7 +26,7 @@ def __get_list(self, subcategories=None, tags=None, content_type=None, with_comm
         if content_type is not None:
             if not isinstance(content_type, list):
                 content_type = [content_type]
-
+            content_type = filter(None, content_type)
```

Comment:
ça fonctionne ça ? pas plutôt `filter(lambda c: c is not None, content_type)` ?

Reply:
J'ai rien dit. https://docs.python.org/2/library/functions.html#filter
“If function is None, the identity function is assumed, that is, all elements of iterable that are false are removed.”
TIL.

Reply:
C'est identique :

>Note that `filter(function, iterable)` is equivalent to the generator expression `(item for item in iterable if function(item))` if function is not `None` and `(item for item in iterable if item)` if function is None.

https://docs.python.org/3/library/functions.html#filter